deploy/kubernetes/examples/demo-secret-configuration-file.json

```diff
@@ -0,0 +1,24 @@
+{

```

Comment:
is `file` really necessary in the filename?

Reply:
the command containing `from-file` so I thought to "connect" it..

Reply:
can said `from-file` command be used to create our regular secret?

Reply:
I dont think so

Reply:
changed